### ISSUE TITLE
Way to minimize build time for exporting image?

### ISSUE THREAD

**Issue by eric1tran** [RPi-Distro/pi-gen#192]
Normal lite build takes ~20 minutes to build. 

README says I can make modifications to a specific stage and build only that. But once I do that, do I need to remove SKIP_IMAGE and build the entire image again?

**Comment by XECDesign** [comment]
Yes, the image needs to be exported each time. I have some ways to make that very fast, but haven't gotten around to doing that yet.
